.gitignore

```diff
@@ -1,3 +1,4 @@
 __pycache__/
-/campusdiscussbackend/__pycache__
+/**/__pycache__
```

Comment:
I think the first line would cover all files of the name `__pycache__`. So this might be unnecessary.